build.gradle.kts

```diff
@@ -75,33 +87,9 @@ config {
 
         people {
             person {
-                id = "pierre"
-                name = "Pierre Leresteux"
-                email = "pierre@saagie.com"
-                roles = listOf("author", "developer")
-            }
-            person {
-                id = "richard"
-                name = "Richard Capraro"
-                email = "richard.capraro@saagie.com"
-                roles = listOf("developer")
-            }
-            person {
-                id = "guillaume"
-                name = "Guillaume Naimi"
-                email = "guillaume.naimi@saagie.com"
-                roles = listOf("developer")
-            }
-            person {
-                id = "igor"
-                name = "Igor Laborie"
-                email = "igor.laborie@saagie.com"
-                roles = listOf("developer")
-            }
-            person {
-                id = "antoine"
-                name = "Antoine Carton"
-                email = "antoine.carton@saagie.com"
+                id = "julien"
```

Comment:
We may remove completely the developpers section as we have done for other GH projects.

Reply:
Not very fair to remove the names of people who have worked on this project, and for some of them on their spare time.

Reply:
done.